en/developer_guide/navigation_panel.rst

```diff
@@ -41,14 +41,65 @@ The menu link must have the following attributes :
 * id : merge of the "nav-" string and the name of the strategy as returned by
   ``NavigationPanelInterface::getName()``, for instance "nav-user"
 
-It can also have data-* attributes that can help you to define custom properties to be used in a Backbone route,
-for example :
+Menu and data table
+~~~~~~~~~~~~~~~~~~~
+
+Many panel entries allow to consult records formated in a data table.
+The columns of this data table can be configured: name of the column, visible or not by default, type of search field...
+The data table view is based on `DataTables library`_.
+So all options coming from this library are allowed in the definition of the columns.
+
+Here is an example of yml data table configuration:
+
+.. code-block:: yaml
+
+    open_orchestra_backoffice.navigation_panel.trashcan.parameters :
+        -
+          name : name
+          title : open_orchestra_backoffice.table.trashcan.name
+          visible : true
+          activateColvis : true
+          searchField : text
+        -
+          name : deleted_at
+          updated_by : name
+          title : open_orchestra_backoffice.table.trashcan.deleted_at
+          visible : true
+          activateColvis : true
+          searchField : date
+
+The parameter ``open_orchestra_backoffice.navigation_panel.trashcan.parameters`` is injected to panel strategy whith a translator.
+The Symfony translator will try to translate each value.
+In this case ``open_orchestra_backoffice.table.trashcan.name`` will be translated.
+
+Here is the service configuration:
+
+.. code-block:: yaml
 
-* data-url : API url to be called to retrieve the data to be displayed
-* data-translated-header : the comma-separated list of column labels in dataTable view (see `Entity list`_)
-* data-visible-elements : the comma-separated list of columns, visible by default and identified by id, in dataTable view (see `Entity list`_)
-* data-displayed-elements : the comma-separated list of all columns identified by id in dataTable view (see `Entity list`_)
-* data-input-header: the comma-separated list of search field of columns (see `Entity list`_)
+    open_orchestra_backoffice.navigation_panel.trashcan:
+        class: "%open_orchestra_backoffice.navigation_panel.administration.class%"
+        arguments:
+            - trashcan
+            - ROLE_ACCESS_DELETED
+            - 250
+            - "%open_orchestra_backoffice.navigation_panel.menu.editorial%"
+            - "%open_orchestra_backoffice.navigation_panel.trashcan.parameters%"
+            - "@translator"
+        calls:
+            - [ setTemplate, [OpenOrchestraBackofficeBundle:BackOffice:Include/NavigationPanel/Menu/Editorial/trashcan.html.twig] ]
+        tags:
+            - { name: open_orchestra_backoffice.navigation_panel.strategy }
+
+Each strategy will define a ``getDatatableParameter`` method to recover the datatable parameters.
+In the common case, the parameters are indexed with the name of the strategy to allow merging without conflict.
```

Comment:
If a documentation doesn't describe everything, it is not usefull, plus who will do it ?

The fact is that you want to say something with this sentence, but I don't get it. So I ask more explanation. Or if this sentence has no importance, you should remove it.